src/validation_workflow/rpm/validation_rpm.py

```diff
@@ -75,6 +68,7 @@ def cleanup(self) -> bool:
         try:
             for project in self.args.projects:
                 execute(f'sudo systemctl stop {project}', ".")
-        except:
-            raise Exception('Failed to Stop Cluster')
+                execute(f'sudo yum remove {project} -y', ".")
+        except Exception as e:
+            raise Exception(f'Exception occurred either while attempting to stop cluster or removing OpenSearch {str(e)}')
```

Comment:
This can be dashboards too right? Either add {project} at class level to show actually what failed or make the message genetic saying `Exception occurred either while attempting to stop cluster or removing OpenSearch/OpenSearch-Dashboards`